plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js

```diff
@@ -124,11 +133,11 @@ const Blueprint = () => {
 					{ error }
 				</Notice>
 			) }
-			<h3>Import Blueprint</h3>
+			<h3>{ __( 'Blueprint', 'woocommerce' ) }</h3>
 			<p>
 				{ createInterpolateElement(
 					__(
-						'Import your blueprint schema to ease the setup process for your store or allow teams to apply each others’ changes. You can import only one Blueprint at the time. Want to know more? <docLink/>',
+						"Blueprints are setup files that contain all the installation instructions, including plugins, themes, and setting. Ease the setup process, allow teams to apply each others' changes and much more. <docLink />",
```

Comment:
Replaced `'` with apostrophes `’`.

```suggestion
						"Blueprints are setup files that contain all the installation instructions, including plugins, themes, and setting. Ease the setup process, allow teams to apply each others’ changes and much more. <docLink />",
```